### PR TITLE
Add Application Data Replication feature in Control Center (to be released in Q1)

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/en/docs/control-center/security/security.md [hunk 1]
```diff
@@ -31,12 +31,26 @@ To disable the digital signing of emails, turn off the toggle. To enable the dig
...
 
-## Single Sign-On
+### Application Data Replication {#application-data-replication}
 
-On the **Single Sign-On** tab, you can set up an identity federation between the Mendix Platform and your corporate identity provider. We call this feature *Bring Your Own Identity Provider (BYOIDP)* and you can find more information in [How to Set Up an SSO (BYOIDP)](/control-center/security/set-up-sso-byoidp/).
+{{% alert color="info" %}}
+This feature is only available for [premium cutomers](/developerportal/deploy/mendix-cloud-deploy/#additional-resources).
+{{% /alert %}}
+
+For security and disaster recovery purposes, you may want to replicate application data in Mendix Cloud to another region. If that is the case, click **Activate** to activate application data replication. By default, application data replication is activated.
```
Comment (Luyao-Zhang-1) [inline content/en/docs/control-center/security/security.md]: Feedback from Leontien:

I would rephrase the text to something like:
This setting is ON by default: Mendix provides a service level agreement for these Cloud Services, which
includes high-availability and disaster recovery measures. These measures involve the replication
of customer’s application data (file storage and database backups) to other availability zones
within the region and to a secondary region to ensure data accessibility in case of an outage in
the primary region. For the most
current information on available Mendix Cloud regions consult the Documentation or visit:
https://www.mendix.com/evaluation-guide/app-lifecycle/mendix-cloud-overview and
https://docs.mendix.com/developerportal/deploy/mxcloudv4#3-where-will-my-data-be-hosted.
If you want to keep your data always in the same region, click Deactivate to deactivate application data replication. You will be asked to sign a statement that you understand this impacts the agreement with Mendix regarding the service levels, disaster recovery and SOC2 compliance.
{{% alert color="info" %}} When you activate or deactivate application data replication, this only affects apps and environments that have not been provisioned yet.  {{% /alert %}}

Comment (Luyao-Zhang-1) [reply content/en/docs/control-center/security/security.md]: You can find context here:
[https://mendix.atlassian.net/wiki/spaces/CloudRnD/pages/3053650007/Application+data+replication+to+other+regions#Text-to-be-shown[…]-waiver-dialogue](https://mendix.atlassian.net/wiki/spaces/CloudRnD/pages/3053650007/Application+data+replication+to+other+regions#Text-to-be-shown-in-the-waiver-dialogue)

Comment (MarkvanMents) [reply content/en/docs/control-center/security/security.md]: Updated in commit ead156a6cf63fb9a2909766875ed257397b197ad

**Review by MarkvanMents — APPROVED** [review]
Ready for publication when feature is released.
